app/src/main/java/com/example/nvdovin/weatherapp/ForecastActivity.java

```diff
@@ -15,9 +19,9 @@
 public class ForecastActivity extends AppCompatActivity implements ForecastView {
     RetrofitFactory retrofitFactory;
     GreenDaoFactory greenDaoFactory;
-
+    RecyclerView recyclerView;
     ForecastPresenter forecastPresenter;
-
+    Integer tempScale = 273;
```

Comment:
Max please pay attention to 
http://www.oracle.com/technetwork/java/codeconventions-135099.html